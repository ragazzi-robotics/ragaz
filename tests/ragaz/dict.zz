def main():
    var a = {'a': 1.0, 'b': 2.0, 'c': 3.0}
    print(a)
    a['z'] = 9
    print(a['z'])

    # TODO: Make this work:
    #a.pop('a')

    for item in a.keys():
        print(item)

    for item in a.values():
        print(item)
