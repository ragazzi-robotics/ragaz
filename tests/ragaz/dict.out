{'a': '1.000000', 'b': '2.000000', 'c': '3.000000', }
9.000000
a
b
c
z
1.000000
2.000000
3.000000
9.000000
